qless-java/src/main/java/com/moz/qless/JobSpec.java

```diff
@@ -0,0 +1,94 @@
+package com.moz.qless;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.moz.qless.client.ClientHelper;
+
+public class JobSpec {
+
+  protected String klass;
```

Comment:
My knee-jerk reaction is that for classes like this don't need them since their purpose in life is really just to be nothing more than a collection of values with no functionality beyond that.

If y'all feel strongly, I'd ad them.